framework/web_app/src/pages/StockOverview.vue

```diff
@@ -1,96 +1,260 @@
-<script setup lang="ts">
-import { StockItem } from 'src/models/StockItem';
```

Comment:
feel free to take this file

Reply:
done, make sure to still gaze over the comments/changes for learning and any mistakes i have made